DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: c14bazAAR
 Title: Download and Prepare C14 Dates from Different Source Databases
-Description: Query different C14 date databases and apply basic data cleaning, merging and calibration steps. Currently available databases: 14cpalaeolithic, 14sea, adrac, austarch, calpal, context, eubar, euroevol, irdd, jomon, kiteeastafrica, medafricarbon, pacea, palmisano, radon, radonb.
-Version: 1.2.0.9000
+Description: Query different C14 date databases and apply basic data cleaning, merging and calibration steps. Currently available databases: 14cpalaeolithic, 14sea, adrac, austarch, calpal, context, emedyd, eubar, euroevol, irdd, jomon, kiteeastafrica, medafricarbon, pacea, palmisano, radon, radonb.
+Version: 1.2.2
```

Comment:
This is still part of our way towards 1.2.1. We will make a new CRAN release soon, don't worry :wink: . For the moment please change back to 1.2.0.9000